packages/wrangler/src/pages/dev.tsx

```diff
@@ -46,6 +47,11 @@ export function Options(yargs: Argv): Argv<PagesDevArgs> {
 				default: true,
 				description: "Run on my machine",
 			},
+			ip: {
+				type: "string",
+				default: "localhost",
```

Comment:
We're gonna default to 0.0.0.0 - https://github.com/cloudflare/wrangler2/issues/1554#issuecomment-1202368103

Can you change wrangler dev too?

```suggestion
				default: "0.0.0.0",
```